sql/core/src/main/scala/org/apache/spark/sql/DataFrame.scala

```diff
@@ -161,11 +161,18 @@ class DataFrame private[sql](
   }
 
   /**
+    * Alias on showString for user convenience
+    */
+  def showString(): String = {
+    showString(20, true);
```

Comment:
actually, since it default to true you can just not provide it, i.e. `showString(20)`